app/controllers/catalog_controller.rb

```diff
@@ -619,4 +620,42 @@ def facet
       format.json
     end
   end
+
+  ##
+  # Render the facet constraints
+  # @param [Hash] localized_params query parameters
+  # @return [String]
+  def render_constraints_filters(localized_params = params)
```

Comment:
This shouldn't be necessary, but I've been really struggling to find a superior solution.  When attempting to invoke this through `super`, I receive the following error:
```
# NameError:
     #   undefined local variable or method `controller' for #<CatalogController:0x00007fcf52ac0b38>
     #   Did you mean?  controller_path
```